ui/src/views/mint-site/mint-site.tsx

```diff
@@ -42,6 +44,31 @@ const initialValues = {
   ens: '',
 } as FormValues;
 
+const OwnerAdress = (props: InputProps) => {
+  const { setFieldValue } = useFormikContext();
+
+  return (
+    <InputGroup size="md">
+      <Input {...props} pr="4.5rem" />
+      <InputRightElement width="4.5rem">
+        <Button
+          h="1.75rem"
+          size="sm"
+          onClick={() => {
+            if (setFieldValue && props.name) {
+              navigator.clipboard
+                .readText()
+                .then((text) => setFieldValue(props.name as string, text));
+            }
+          }}
```

Comment:
Let's avoid adding functions in the middle of the component render to improve readability.